scripts/issue/23/scrape_tokens.py

```diff
@@ -0,0 +1,39 @@
+from brownie import config
+import requests
+import pandas as pd
+
+def main():
+    """
+    Get all ERC-20 tokens and their balances in a csv for a given address with ethplorer api
+    
+    `scrape` can be called directly from command line with an address:
+    brownie run scrape_tokens.py scrape <address>
+    
+    edit `wallet_address` if you are running from brownie console
+    """
+    
+    wallet_address = ''
+    scrape(wallet_address)
+
+def scrape(address):
+    with requests.session() as session:
+        ethplorer_key = config['keys']['ETHPLORER_API_KEY']
+        api_key = ethplorer_key if ethplorer_key else 'freekey'
+        res = session.get(
+            f'https://api.ethplorer.io/getAddressInfo/{address}?showETHTotals=false', 
+            params={"apiKey": api_key}
+            )
+        assert res.status_code == 200
```

Comment:
we rather not rely on external apis if not needed; a `interface.ERC20(token_addr).balanceOf(addr)` would be a lot more reliable (as balances are all taken verifiable directly from one and the same forked block). personally i would loop over `registry.eth.treasury_tokens`, although i know that that is not fully waterproof in the sense of being all inclusive..